src/Microsoft.OData.Edm/Vocabularies/Annotations/EdmVocabularyAnnotation.cs

```diff
@@ -44,8 +47,67 @@ public EdmVocabularyAnnotation(IEdmVocabularyAnnotatable target, IEdmTerm term,
             this.term = term;
             this.qualifier = qualifier;
             this.value = value;
+            this.usesDefault = false;
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the EdmVocabularyAnnotation class
+        /// to the default value.
+        /// </summary>
+        /// <param name="target">Element the annotation applies to.</param>
+        /// <param name="term">Term bound by the annotation</param>
+        public EdmVocabularyAnnotation(IEdmVocabularyAnnotatable target, IEdmTerm term)
+            :this(target, term, "")

```

Comment:
>"" [](http://example.com/codeflow?start=32&length=2)

null, not empty string

Reply:
Null was my first thought as well, but then the constructor call is ambiguous with the other three-constructor method and I therefore get an error. I opted for "" instead because it's string-specific, so the computer can tell which constructor I'm intending. If you have a different suggestion, that would definitely help!

Reply:
Done ✅

Reply:
can you do
         :this(target, term, (string)null)
?